src/bridgestan.h

```diff
@@ -219,7 +219,7 @@ int log_density_gradient(model_rng* mr, bool propto, bool jacobian,
  * @param[in] propto `true` to discard constant terms
  * @param[in] jacobian `true` to include change-of-variables terms
  * @param[in] theta unconstrained parameters
- * @param[out] lp log density to be set
+ * @param[out] val log density to be set
```

Comment:
Good catch to align doc with the function. 

Isn't it `lp` in all other functions?  Anyway, this is something to double check later.

Reply:
It's called `lp` for log_density and `val` for the gradient and Hessian functions. Agreed would be good to do a pass on later, for now it was just generating a doxygen warning